src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VideoFanTakeover.java

```diff
@@ -197,4 +201,10 @@ public boolean isTimeProgressing(double quartileTime, double midTime) {
             "Video time is not progressing, quartileTime " + quartileTime + " is not smaller than midTime " + midTime , false, driver);
     return false;
   }
+//  This scroll has been implemented because driver was not able to execute script in JavascriptActions
+  private void scrollToAdsElement(WebElement element, int offset) {
```

Comment:
could you use here the method from https://github.com/Wikia/selenium-tests/pull/1644/files#diff-bbc0da1a40c1de89845986678abe1ce1R114 (jsActions class)